tests/smoke/coc.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+SMOKE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+. ${SMOKE_DIR}/base.sh code_of_conduct "Citizen Code Of Conduct Suitable for large communities and events"
```

Comment:
I think you need to use the exactly name you define here: https://github.com/osl-incubator/cookiecutter-python/pull/62/files#diff-c2067af38dcc45bd90a9c6b3ef58f6995ac2e7386dd61e802911b3deff05908eR31